rules-tests/TypeDeclaration/Rector/ClassMethod/NumericReturnTypeFromStrictScalarReturnsRector/Fixture/binary_shift.php.inc

```diff
@@ -6,7 +6,12 @@ final class BinaryShift
 {
     public function resolve(int $first, int $second)
     {
-        return $first & $second;
+        return $first << $second;
```

Comment:
since the `BinaryShift` name got free'ed by moving the `BitwiseAnd` into a same named file, this fixture now also makes more sense